tools/src/main/java/org/apache/kafka/trogdor/common/Platform.java

```diff
@@ -47,6 +51,7 @@ public static Platform parse(String curNodeName, String path) throws Exception {
             String platformName = platformNode.textValue();
             return Utils.newParameterizedInstance(platformName,
                 String.class, curNodeName,
+                Time.class, SystemTime.SYSTEM,
```

Comment:
I see a unit test failure because these parameters don't match the constructor of `BasicPlatform`.